src/ProjectSystemTools/BuildLogging/Model/RpcContracts/IBuildLoggerService.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.VisualStudio.ProjectSystem.Tools.Providers;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model.RpcContracts
+{
+    /// <summary>
+    /// The main interface between the client and the server.
+    /// These operations are async
+    /// </summary>
+    public interface IBuildLoggerService
+    {
+        /// <summary>
+        /// Returns whether or not the build logging window is currently tracking logs or not
+        /// </summary>
+        /// <returns>True if build logging window is tracking logs and false otherwise</returns>
+        bool IsLogging();
```

Comment:
This feels like it should be a property rather than a method.

Reply:
I was thinking I would need this to be a method once I introduced the Codespaces side of things,
It does make sense for it to be a property though, I'll look into if I can send a property over server client connection.

I'll make these properties for now

Reply:
You're right. This will need to be an `IsLoggingAsync` method when this becomes an RPC service interface. No need to change it to a property now.